cristin-biobank/src/main/java/no/unit/nva/biobank/fetch/FetchBiobankHandler.java

```diff
@@ -0,0 +1,46 @@
+package no.unit.nva.biobank.fetch;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.HttpURLConnection;
+import no.unit.nva.biobank.client.CristinBiobankApiClient;
+import no.unit.nva.biobank.model.QueryBiobank;
+import no.unit.nva.biobank.model.nva.Biobank;
+import no.unit.nva.cristin.common.handler.CristinHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+
+public class FetchBiobankHandler extends CristinHandler<Void, Biobank> {
+
+    private final transient CristinBiobankApiClient cristinClient;
+
+    @JacocoGenerated
+    public FetchBiobankHandler() {
+        this(CristinBiobankApiClient.defaultClient(), new Environment());
+    }
+
+    public FetchBiobankHandler(CristinBiobankApiClient cristinClient, Environment environment) {
+        super(Void.class, environment);
+        this.cristinClient = cristinClient;
+    }
+
+    @Override
+    protected Biobank processInput(Void input, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        var query = (QueryBiobank)
+                        QueryBiobank.builder()
+                            .fromRequestInfo(requestInfo).build();
+        context.getLogger()
```

Comment:
Not sure if you should use this, im mnore familiar with the other way to use logger. `Logger.getLogger(MyClass.class).info(xxx)`

Reply:
Thanks, I had overlooked this one!